components/src/core/drawer/drawer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { DrawerInheritableProps, inheritDrawerProps } from './inheritable-types';
+import * as Colors from '@pxblue/colors';
+import MatIcon from 'react-native-vector-icons/MaterialIcons';
+
+const styles = StyleSheet.create({
+    container: {
+        backgroundColor: Colors.white[50],
+        zIndex: 2,
+        flex: 1,
+        height: '100%',
+    },
+});
+
+export const Drawer: React.FC<DrawerInheritableProps> = (props) => {
+    // Set theme-related default props here.
+    const activeItemBackgroundColor = Colors.blue[50];
+    const findChildByType = useCallback(
+        (type: string): JSX.Element[] =>
+            React.Children.map(props.children, (child: any) => {
+                if (child && child.type) {
+                    const name = child.type.displayName;
+                    if (name && name.includes(type)) {
+                        return child;
+                    }
+                }
+            }) || [],
+        [props]
+    );
+
+    const getSectionByDisplayName = useCallback(
+        (displayName: string, inherit = false): JSX.Element[] =>
+            findChildByType(displayName)
+                .slice(0, 1)
+                .map((child) => {
+                    let inheritableProps = {};
+                    if (inherit) {
+                        inheritableProps = inheritDrawerProps({ ...props, activeItemBackgroundColor }, child.props);
+                    }
+                    return React.cloneElement(child, inheritableProps);
+                }),
+        [props]
+    );
+
+    return (
+        <View style={styles.container}>
+            {getSectionByDisplayName('DrawerHeader')}
+            {getSectionByDisplayName('DrawerSubheader')}
+            {getSectionByDisplayName('DrawerBody', true)}
+            {getSectionByDisplayName('DrawerFooter')}
+        </View>
+    );
+};
+
+Drawer.defaultProps = {
+    activeItemBackgroundShape: 'round',
+    activeItemFontColor: Colors.blue[500],
+    activeItemIconColor: Colors.blue[500],
```

Comment:
default colors should come from the theme. Take a peek at the `useTheme` function from react-native-paper. I have a couple examples of doing this and using the theme to change styles in the workflows repo: https://github.com/pxblue/react-native-workflows/blob/dev/login-workflow/src/components/ResizingClearButton.tsx#L64

Reply:
I've updated the default `activeItemFontColor` and `activeItemIconColor` props to use theme colors.  We dont have any theme value that should be `activeItemBackgroundColor` of Blue[50], so I kept the Blue[50] hardcoded as the default.